guides/common/modules/con_puma-tunings.adoc

```diff
@@ -3,5 +3,5 @@
 [id="Puma_Tunings_{context}"]
 = Puma tunings
 
-Puma is a ruby application server which is used for serving the Foreman related requests to hosts.
+Puma is a ruby application server which is used for serving all Foreman related requests to hosts.
```

Comment:
Should this still say "Foreman"?

Reply:
... and now I read the PR's description. I'm still unsure about this. Is there a way to rephrase the sentence a bit more?

```suggestion
Puma is a ruby application server which is used for serving all requests to hosts in a {Project} deployment.
```

Reply:
IMO yes because it's about the component, not the downstream product/open source project.

Reply:
At the very least, it looks strange right next to "Satellite" which is used in the next sentence. That's why I'd prefer rewording if possible, to avoid mentioning the component. However, not a blocker if you're confident about this :)

Reply:
Thanks for indulging my curiosity! :) Approved.